core/src/spotify_id.rs

```diff
@@ -133,3 +255,185 @@ impl fmt::Display for FileId {
         f.write_str(&self.to_base16())
     }
 }
+
+#[inline]
```

Comment:
It's my understanding that `#[inline]` doesn't do anything unless the function is used across crates without LTO (which it can't be since it's not `pub`). Is there any point of it here?